tests/ui/parser/f16-f128.rs

```diff
@@ -0,0 +1,37 @@
+// Make sure we don't ICE while incrementally adding f16 and f128 support
```

Comment:
I don't think parser is quite right, the parser doesn't care :)

Reply:
Yeah, this test does literally nothing lol

Reply:
There didn't seem to be a good folder :) I can move it if there is a better place

I know it does nothing but I just wanted a test that can be updated as the steps move along